src/components/v5/common/CompletedAction/partials/PaymentBuilder/partials/PaymentBuilderWidget/PaymentBuilderWidget.tsx

```diff
@@ -161,14 +161,33 @@ const PaymentBuilderWidget: FC<PaymentBuilderWidgetProps> = ({ action }) => {
     },
   ];
 
+  const cancelItems: StepperItem<ExpenditureStep>[] = [
+    {
+      key: ExpenditureStep.Create,
+      heading: { label: formatText({ id: 'expenditure.createStage.label' }) },
+      content: (
+        <FinalizeWithPermissionsInfo userAdddress={expenditure?.ownerAddress} />
+      ),
+    },
+    {
+      key: ExpenditureStep.Cancel,
+      heading: { label: formatText({ id: 'expenditure.cancelStage.label' }) },
+      content: (
+        <FinalizeWithPermissionsInfo userAdddress={expenditure?.ownerAddress} />
+      ),
+    },
+  ];
```

Comment:
This works for now as it's the "Before locked state" scenario, but the UI will have to show all the steps that might have happened before cancelling: 
<img width="325" alt="image" src="https://github.com/JoinColony/colonyCDapp/assets/112586815/a8c4563b-d9f2-4ef0-8494-68a108fb3fc3">

So it will be necessary to show and hide each step individually depending on whether the expenditure is cancelled.

Reply:
This PR is focused only on `before Locked state`. I've changed logic a bit so it will be easier to handle in `after locked state`